Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int[] arr = new int[]{1, 2, 3, 4, 5};
+        System.out.println("Even numbers:");
+
+        for (int i = 0; i < arr.length; i = i * 2)

```

Comment:
**Suggestion**: Modify the loop condition to ensure it iterates correctly over even indices without causing an ArrayIndexOutOfBoundsException. Consider using 'i += 2' instead of 'i = i * 2' to iterate through even indices. <br> **Code Suggestion**: 
 ```
 -        for (int i = 0; i < arr.length; i = i * 2)
 +        for (int i = 1; i < arr.length; i += 2)
 ```